src/shared/sam/cli/samCliInvokerUtils.ts

```diff
@@ -9,9 +9,24 @@ import { SpawnOptions } from 'child_process'
 import { BasicLogger, getLogger } from '../../logger'
 import { ChildProcessResult } from '../../utilities/childProcess'
 
+export interface SamCliProcessInvokeSettings {
```

Comment:
Replace `Settings` with `Parameters` or `Options`. Prefer `Parameters` for consistency with the rest of the codebase, unless you have a good reason for an exception.